### DISCUSSION TITLE
call #wg-saft-ao: 2022-11-30

### DISCUSSION THREAD

**Discussion by cryptolopes** [assoft-portugal/SAF-T-AO#208] [Announcements]
#### ⚽ Objetivo da reunião:

- Apresentação de dados estatísticos da AGT
- Antecipação dos webservices para a comunicação de faturas e documentos equivalentes
- Questões da comunidade

<hr>
Estava prevista a análise de caso de problemas identificados pela AGT relacionados com a inconsistência de informação entre o documento original e o reporte de SAF-T (AO) efetuado pelos contribuintes, mas não foi possível. Este tema ficará para outras reuniões.
<hr>

#### ⌚ Hora de início: 
14.30 GMT

#### 📄 Notas:

- O número de ficheiros SAF-T (AO) rejeitados pela AGT tem vindo a decrescer com o tempo. Pese embora seja uma tendência já esperada, atendendo aos inúmeros esforços de todas as partes para a implementação deste projeto, espera-se que continue em sentido decrescente, o que se traduzirá num cada vez menor número de erros e de melhor informação por parte dos contribuintes. Um aspeto positivo, tem que ver com as razões para a rejeição de ficheiros. De acordo com a AGT, a maioria das rejeições está relacionada com a tentativa de submissão de ficheiros por aplicações não validadas e também pela tentativa de envio de ficheiros em duplicado.
![image](https://user-images.githubusercontent.com/32396185/206230018-b8de31f5-7bca-4108-a076-fd9a70f136ec.png)

- A comunicação dos elementos das faturas por Webservice vai começar a ser disponibilizada muito em breve. Para além da documentação técnica, os fabricantes de software irão ter à sua disposição um ambiente de qualidade onde podem realizar testes de integração com os serviços da AGT.

#### ❔ Questões:

<html>
<body>
<div class="clickup-table-view">

Pergunta | Resposta
-- | --
Como proceder para renovar o certificado de validação de software quando em causa existir um processo de fusão de empresas ou de aquisição das participações sociais de uma empresa por parte de outra? | A entidade deve entrar em contacto com a AGT. Haverá casos em que será possível manter o mesmo número de validação e outros casos em que terão que ser iniciados novos processos de validação. Para um número significativo de casos, é possível proceder à transferência da titularidade do certificado sem necessidade de nova revalidação.
É necessário revalidar o certificado sempre que sejam introduzidas novas funcionalidades na aplicação de faturação? | Não. Porém, o fabricante deve ter em consideração que as novas funcionalidades têm de cumprir com as regras e a legislação aplicável.
Como proceder à anulação de recibos? | Através do método de anulação `PaymentStatus` com o valor `A`. Emissão de novo recibo com o valor correspondente à(s) linha(s) em sentido contrário, ou seja, `DebitAmount` caso o original tenha sido emitido como `CreditAmount`.

</div>
</body>
</html>

#### 👷 Action items:

<html>
<body>
<div class="clickup-table-view">

Objetivos | Atribuído a
-- | --
Trabalhar o caso dos documentos modificados | #wg-saft-ao
  |  
  |  

</div>
</body>
</html>

#### ⏭️ Próxima reunião:
2022-12-15 14.30 GMT

#### 📁 SAF-T AO

[![GitHub All Releases](https://img.shields.io/github/downloads/assoft-portugal/SAF-T-AO/total)](https://github.com/assoft-portugal/SAF-T-AO/releases) [![GitHub issues](https://img.shields.io/github/issues-raw/assoft-portugal/SAF-T-AO)](https://github.com/assoft-portugal/SAF-T-AO/issues) [![GitHub tag (latest SemVer)](https://img.shields.io/github/v/tag/assoft-portugal/SAF-T-AO)](https://github.com/assoft-portugal/SAF-T-AO/releases) [![License](https://img.shields.io/badge/license-MIT-blue.svg)](https://github.com/assoft-portugal/SAF-T-AO/blob/master/LICENSE) [![Discussion](https://img.shields.io/badge/discusion-telegram-blue)](https://t.me/saftao)

**Comment by mt-m2g** [comment]
Boa tarde senhor Nelson!
 Tem alguma forma de entrar em contacto consigo, ou seja se possivél colocar o seu email para entrar em contacto.
 
 Saudações!

**Reply by cryptolopes** [reply]
Pode ser por aqui.
